src/webrtc/callFeed.ts

```diff
@@ -295,6 +298,12 @@ export class CallFeed extends TypedEventEmitter<CallFeedEvent, EventHandlerMap>
             this.analyser = null;
             releaseContext();
         }
+        this.disposed = true;
+        this.emit(CallFeedEvent.Disposed, this.disposed);
+    }
+
+    public idDisposed(): boolean {
```

Comment:
I think this would be a good use case for the public getter + private setter pattern, i.e.

```typescript
private _disposed = false;
public get disposed(): boolean {
    return this._disposed;
}
private set disposed(value: boolean) {
    this._disposed = value;
}